compiler/src/dmd/func.d

```diff
@@ -258,22 +258,29 @@ extern (C++) struct Ensure
 
 }
 
-/***********************************************************
- */
-extern (C++) class FuncDeclaration : Declaration
+// Most function don't have contracts, so save memory by grouping
+// this information into a separate struct
+private struct ContractInfo
 {
     Statements* frequires;              /// in contracts
     Ensures* fensures;                  /// out contracts
     Statement frequire;                 /// lowered in contract
     Statement fensure;                  /// lowered out contract
-    Statement fbody;                    /// function body
-
-    FuncDeclarations foverrides;        /// functions this function overrides
     FuncDeclaration fdrequire;          /// function that does the in contract
     FuncDeclaration fdensure;           /// function that does the out contract
-
     Expressions* fdrequireParams;       /// argument list for __require
     Expressions* fdensureParams;        /// argument list for __ensure
+}
+
+/***********************************************************
+ */
+extern (C++) class FuncDeclaration : Declaration
```

Comment:
This diff looks unnecessary. The class declaration should be unaffected

Reply:
Github is just being weird here, I didn't modify the class definition.